web/viewer.html

```diff
@@ -147,6 +147,32 @@
           </div>
         </div>  <!-- findbar -->
 
+        <div class="editorParamsToolbar hidden doorHangerRight" id="editorFreeTextParamsToolbar">
+          <div class="editorParamsToolbarContainer">
+            <div class="editorParamsSetter">
+              <label for="fontColor" class="editorParamsLabel" data-l10n-id="editor_free_text_font_color">Font Color</label>
+              <input type="color" id="editorFreeTextColor" name="fontColor" class="editorParamsColor" tabindex="100">
+            </div>
+            <div class="editorParamsSetter">
+              <label for="fontSize" class="editorParamsLabel" data-l10n-id="editor_free_text_font_size">Font Size</label>
+              <input type="range" id="editorFreeTextFontSize" class="editorParamsSlider" name="fontSize" value="10" min="5" max="100" step="1" tabindex="101">
+            </div>
+          </div>
+        </div>
+
+        <div class="editorParamsToolbar hidden doorHangerRight" id="editorInkParamsToolbar">
+          <div class="editorParamsToolbarContainer">
+            <div class="editorParamsSetter">
+              <input type="color" id="editorInkColor" name="inkColor" class="editorParamsColor" tabindex="102">
+              <label for="color" class="editorParamsLabel" data-l10n-id="editor_ink_line_color">Line Color</label>
+            </div>
+            <div class="editorParamsSetter">
+              <input type="range" id="editorInkThickness" class="editorParamsSlider" name="lineThickness" value="1" min="1" max="20" step="1" tabindex="103">
+              <label for="lineThickness" class="editorParamsLabel" data-l10n-id="editor_ink_line_thickness">Line Thickness</label>
```

Comment:
Let's flip the order of these elements as well, for consistency with the FreeText-toolbar.